python_http_client/exceptions.py

```diff
@@ -0,0 +1,53 @@
+class HTTPError(Exception):
+	''' Base of all other errors'''
+	def __init__(self,error):
+		self.code = error.code
+		self.reason = error.reason
+		#self.headers = error.headers
```

Comment:
it does. you have to read docs and look at the source.

HTTPError is based on URLError which has the following attributes

```python
err.url
err.code
err.hdrs
# to get the body
err.read()

```